src/site/_includes/components/Author.js

```diff
@@ -41,12 +41,14 @@ module.exports = ({
   const fullName = `${author.name.given} ${author.name.family}`;
   return html`
     <div class="w-author">
-      <img
-        class="w-author__image ${small && `w-author__image--small`}"
-        src="/images/authors/${avatar}.jpg"
-        alt="${fullName}"
-      />
-      ${AuthorInfo({post, author, showSocialMedia})}
+      <a href="/authors/${id}" tabindex="-1">
```

Comment:
Is there a reason people can't tab to us? :(

Reply:
ah that was my mistake. I think I was trying to cut down on tab targets within the card since the image, headline, and body text all link to the same resource.

but yeah, we should remove the tabindex here because authors link to a completely unique url and that needs to be keyboard accessible.